packages/ubuntu_init/integration_test/e2e_test.dart

```diff
@@ -29,6 +29,11 @@ void main() {
     await tester.tapNext();
     await tester.pumpAndSettle();
 
+    await tester.testAccessibilityPage();
+    await tester.tapNext();
+    await tester.pumpAndSettle();
+    // TODO: Test accessibility settings
```

Comment:
Just like for the other gsettings-based services proper assertions are still missing here, as we haven't found a working solution for the CI yet.
However, this test ensures that the corresponding endpoint in provd responds successfully.